docs/tutorial.rst

```diff
@@ -385,6 +351,57 @@ There may be times when you need to import an existing scratch org that wasn't c
 
 You can hop into a browser logged into any org in your keychain with `cci org browser <org_name>`.
 
+Creating a Connected App
+------------------------
+
+In order to connect persistent orgs such as a Developer Edition, Enterprise Edition, or Sandbox org to CumulusCI, you need to have a Connected App configured in a persistent Salesforce org.
+
+Create With CumulusCI
+^^^^^^^^^^^^^^^^^^^^^
+
+CumulusCI includes a task to easily deploy the Salesforce Connected App to any org in your sdfx keychain.  By default, this will deploy to the org configured as the defaultdevhubusername.
+
+.. code-block:: console
+
+    $ cci task run connected_app
+
+This command will also configure CumulusCI's connected_app service in the keychain for you.  If you want to see the information for the connected app, you can view it with:
```

Comment:
Not unless they specify `-o connect True`, right?

Reply:
The default cumulusci.yml sets this to True.  The overwrite option is not defaulted to True though so it will avoid overwriting your existing connected_app if it's already configured.